.github/ISSUE_TEMPLATE/certification_request.yaml

```
@@ -0,0 +1,57 @@
+name: Request for CAPI Provider Certification
+description: Request that a CAPI provider is added to the list of certified providers.
+body:
+  - type: textarea
+    id: provider
+    attributes:
+      label: What is the provider you would like to be added?
+      placeholder: "Specify the name and the type of provider."
+    validations:
+      required: true
+
+  - type: textarea
+    id: repository
+    attributes:
+      label: What is the repository URL?
+      placeholder: "Link to GitHub repository where the provider's source code is hosted."
+    validations:
+      required: true
+
+  - type: textarea
+    id: check_successful
+    attributes:
+      label: Did you follow the certification process?
+      placeholder: "You need to follow the certification process before requesting a provider is added to the list. You can refer to the Turtles documentation."
```

```suggestion
      placeholder: "You need to follow the certification process before requesting a provider to be added to the list. You can refer to the Turtles documentation."
```

Would it be better also to refer to the [cert process doc](https://github.com/rancher/turtles-docs/pull/149/files#diff-7992c5db33171de1a0e17b30f25c4c400a5cc3fa3063231af9c39e8eef5531eb) directly in the second sentence once?